src/components/shared/pick-track/index.tsx

```diff
@@ -1,4 +1,6 @@
 import * as React from 'react';
+import { getFirebase } from '../../../../firebase-functions/firebase';
+import * as moment from 'moment';
```

Comment:
only thing here would be to remove this library, because it's very heavy for frontend usage and we can use the JavaScript API. It's a good idea to use it in the backend if we wanted though, because it does make data handling easier

Reply:
Ok I'll see how to make it work with javascript API. I used 'moment.js' as I needed to extract the date from the firestore time stamp